build-static.sh

```diff
@@ -117,88 +117,82 @@ if [ -n "${CLEAN}" ]; then
 	go clean -cache
 fi
 
-cache_key="${PHP_VERSION}-${PHP_EXTENSIONS}-${PHP_EXTENSION_LIBS}"
-
-# Build libphp if necessary
-if [ -f dist/cache_key ] && [ "$(cat dist/cache_key)" = "${cache_key}" ] && [ -f "dist/static-php-cli/buildroot/lib/libphp.a" ]; then
-	cd dist/static-php-cli
+mkdir -p dist/
+cd dist/
 
-	if [ -f "./spc" ]; then
-		spcCommand="./spc"
-	elif [ -f "bin/spc" ]; then
-		spcCommand="./bin/spc"
+if type "brew" >/dev/null 2>&1; then
+	if ! type "composer" >/dev/null; then
+		packages="composer"
 	fi
-else
-	mkdir -p dist/
-	cd dist/
-	echo -n "${cache_key}" >cache_key
-
-	if type "brew" >/dev/null 2>&1; then
-		if ! type "composer" >/dev/null; then
-			packages="composer"
-		fi
-		if ! type "go" >/dev/null 2>&1; then
-			packages="${packages} go"
-		fi
-		if [ -n "${RELEASE}" ] && ! type "gh" >/dev/null 2>&1; then
-			packages="${packages} gh"
-		fi
-
-		if [ -n "${packages}" ]; then
-			# shellcheck disable=SC2086
-			brew install --formula --quiet ${packages}
-		fi
+	if ! type "go" >/dev/null 2>&1; then
+		packages="${packages} go"
 	fi
-
-	if [ "${SPC_REL_TYPE}" = "binary" ] && [[ ! "${arch}" =~ arm ]]; then
-		mkdir -p static-php-cli/
-		cd static-php-cli/
-		curl -o spc -fsSL "https://dl.static-php.dev/static-php-cli/spc-bin/nightly/spc-linux-${arch}"
-		chmod +x spc
-		spcCommand="./spc"
-	elif [ -d "static-php-cli/src" ]; then
-		cd static-php-cli/
-		git pull
-		composer install --no-dev -a
-		spcCommand="./bin/spc"
-	else
-		git clone --depth 1 https://github.com/crazywhalecc/static-php-cli --branch main
-		cd static-php-cli/
-		composer install --no-dev -a
-		spcCommand="./bin/spc"
+	if [ -n "${RELEASE}" ] && ! type "gh" >/dev/null 2>&1; then
+		packages="${packages} gh"
 	fi
 
-	# extensions to build
-	if [ -z "${PHP_EXTENSIONS}" ]; then
-		# enable EMBED mode, first check if project has dumped extensions
-		if [ -n "${EMBED}" ] && [ -f "${EMBED}/composer.json" ] && [ -f "${EMBED}/composer.lock" ] && [ -f "${EMBED}/vendor/installed.json" ]; then
-			cd "${EMBED}"
-			# read the extensions using spc dump-extensions
-			PHP_EXTENSIONS=$(${spcCommand} dump-extensions "${EMBED}" --format=text --no-dev --no-ext-output="${defaultExtensions}")
-		else
-			PHP_EXTENSIONS="${defaultExtensions}"
-		fi
-	fi
-	# additional libs to build
-	if [ -z "${PHP_EXTENSION_LIBS}" ]; then
-		PHP_EXTENSION_LIBS="${defaultExtensionLibs}"
+	if [ -n "${packages}" ]; then
+		# shellcheck disable=SC2086
+		brew install --formula --quiet ${packages}
 	fi
-	# The Brotli library must always be built as it is required by http://github.com/dunglas/caddy-cbrotli
-	if ! echo "${PHP_EXTENSION_LIBS}" | grep -q "\bbrotli\b"; then
-		PHP_EXTENSION_LIBS="${PHP_EXTENSION_LIBS},brotli"
+fi
+
+if [ "${SPC_REL_TYPE}" = "binary" ]; then
```

Comment:
I'm not entirely sure why they previous commit limited to this != aarch64. There is a spc aarch64 binary available to download and it was working before, so I reverted this.

Reply:
If ${arch} is `arm64`, then it tries to download `spc-linux-arm64` which doesn't exist (since the binary name is spc-linux-aarch64)

Reply:
I see, thanks. Fixing it!

Reply:
@pierredup Wait, shouldn't we just generally set $arch to aarch64 if it's arm? Otherwise it would also build `frankenphp-linux-arm64`, which would not be expected.

Reply:
For Linux, yes that makes sense. Mac should still be `frankenphp-mac-arm64` though.

Reply:
I'm trying to re-create the scenario that caused me to run into this, can't remember the steps exactly, but I was using the `dunglas/frankenphp:static-builder-gnu` image and trying to create an arm64 build. Will see if I can replicate the steps again

Reply:
Centos 7 reports aarch64, you can see it with the -gnu release.

Reply:
Oh if macos says arm64 that will be where you ran into it. The macos binary is also called aarch64.